include/pistache/net.h

```diff
@@ -65,6 +64,7 @@ class AddrInfo {
 class Port {
 public:
     Port(uint16_t port = 0);
```

Comment:
All single-argument constructors should be explicit.  Please fix this one also.

Reply:
@dennisjenkins75 Yes, you are right, but in some cases it's ok.
Adding `explicit` to this contructor doesn't permit us to write the following code:
```
port_ = HTTP_STANDARD_PORT;
port = std::stol(argv[1]);
```
What do you think about it?

Reply:
hmm... idk.  I typically add it b/c cppcheck will complain if I don't. :)